ibc-core/ics02-client/context/src/client_state.rs

```diff
@@ -172,14 +186,23 @@ where
         client_message: Any,
     ) -> Result<(), ClientError>;
 
-    // Update the client state and consensus state in the store with the upgraded ones.
+    /// Update the client state and consensus state in the store with the upgraded ones.
     fn update_state_on_upgrade(
         &self,
         ctx: &mut E,
         client_id: &ClientId,
         upgraded_client_state: Any,
```

Comment:
Hmm. it seems a convention for using `Any` over `Self`.

Reply:
Yea, `Any` is used pretty consistently throughout the codebase instead of `Self`; I'm going to stick with that for now.